src/test/components/MarkerTooltipContents.test.js

```diff
@@ -234,18 +234,21 @@ describe('MarkerTooltipContents', function() {
     const threadIndex = getSelectedThreadIndex(state);
     const tracingMarkers = selectedThreadSelectors.getTracingMarkers(state);
 
-    tracingMarkers.forEach(marker => {
-      expect(
-        renderer.create(
-          <Provider store={store}>
-            <MarkersTooltipContents
-              marker={marker}
-              threadIndex={threadIndex}
-              className="propClass"
-            />
-          </Provider>
-        )
-      ).toMatchSnapshot();
-    });
+    expect(
+      renderer.create(
+        <Provider store={store}>
+          <Fragment>
+            {tracingMarkers.map((marker, i) => (
+              <MarkersTooltipContents
+                key={i}
+                marker={marker}
+                threadIndex={threadIndex}
+                className="propClass"
+              />
+            ))}
+          </Fragment>
+        </Provider>
+      )
+    ).toMatchSnapshot();
```

Comment:
I changed this so that it's easier to see the changes to the snapshot. See the first commit for the test changes and the snapshot changes for master, and the last commit for the snapshot changes for this patch.

Reply:
Nice, more signal, less noise. Also this is helpful as a separate commit.